Mathlib/Geometry/RingedSpace/PresheafedSpace/HasColimits.lean

```diff
@@ -248,7 +248,9 @@ def colimitCoconeIsColimit (F : J ⥤ PresheafedSpace.{_, _, v} C) :
       m.base =
         colimit.desc (F ⋙ PresheafedSpace.forget C) ((PresheafedSpace.forget C).mapCocone s) := by
       dsimp
-      ext j
+      -- `colimit.hom_ext` used to be automatically applied by `ext`
```

Comment:
Used to be when? Let's add a reference to the PR number in case someone is digging.

Reply:
```suggestion
      -- `colimit.hom_ext` used to be automatically applied by `ext` before https://github.com/leanprover-community/mathlib4/pull/21302
```